apps/vscode-extension/src/intellisense/configuration.ts

```diff
@@ -43,16 +43,36 @@ const getConfigurationTagsFromEslintConfig = (eslintConfig: any, comment: string
   const widgetParamsToPropose = Object.keys(widgetConfig).filter((paramName) => !paramsPresent.has(paramName));
 
   return {
+    tags: {
```

Comment:
there is no risk of conflict if the snippet already exist elsewhere with the same name ?

Reply:
It will override the previous one 
maybe in another PR we can think about adding these tags only when we are in config interface or property comment